scripts/createPlaygroundBuild.mjs

```diff
@@ -6,7 +6,7 @@
 
 // This script does two things: 
 //  
-//  - Listens to changes to the built version of TypeScript (via a filewatcher on `built/local/typescriptServices.js`)
+//  - Listens to changes to the built version of TypeScript (via a filewatcher on `built/local/typescript.js`)
```

Comment:
This will have to be updated to match the final state of monaco-editor. e.g. https://github.com/microsoft/monaco-editor/pull/3356